company/strategy.md

```diff
@@ -16,21 +38,85 @@ To achieve our purpose, our mission is to give every organization the ability to
 
 Our [values](values.md) are the principles we want to uphold as we work toward our mission.
 
+<!-- TODO: these will change, see https://docs.google.com/document/d/1_L-XOT4oPTOuUkT4_JMykneo_wPn2WqC2BXR6_-U5dI/edit -->
+
 - [People](values.md#people)
 - [Journey](values.md#journey)
 - [Progress](values.md#progress)
 
 ## Vision
 
-With Sourcegraph:
+### 1-year vision {#1-year-vision}
+
+Sourcegraph universal code search is the one place to find and fix things across all code.
+
+- Sourcegraph knows most everything that can be known about your code (from the compiler, build systems, code hosts, runtime metrics, docs, issues, project management, etc.).
+- It's easy to find and use this information on Sourcegraph itself and in your editor, code host, and code reviews.
+- You use Sourcegraph in your existing workflows for writing, reviewing, understanding, and fixing code.
+
+### 3-year vision {#3-year-vision}
+
+Sourcegraph understands higher-level software development patterns and manages processes proactively, which lets you spend more time on the higher-leverage tasks that truly require your human brain.
```

Comment:
The "proactive" bit is still fuzzy to me. Is Sourcegraph being proactive or does it help devs be proactive?

Reply:
reworded for clarity